ui/apps/platform/src/routePaths.js

```diff
@@ -91,6 +91,13 @@ export const vulnManagementPendingApprovalsPath = `${vulnManagementRiskAcceptanc
 export const vulnManagementApprovedDeferralsPath = `${vulnManagementRiskAcceptancePath}/approved-deferrals`;
 export const vulnManagementApprovedFalsePositivesPath = `${vulnManagementRiskAcceptancePath}/approved-false-positives`;
 
+// VM 2.0 "Vulnerabilities" paths
+export const vulnerabilitiesBasePath = `${mainPath}/vulnerabilities`;
+export const vulnerabilitiesWorkloadCvesPath = `${vulnerabilitiesBasePath}/workload-cves`;
+export const vulnerabilitiesWorkloadCveSinglePath = `${vulnerabilitiesBasePath}/workload-cves/cve/:cveId`;
+export const vulnerabilitiesWorkloadCveImageSinglePath = `${vulnerabilitiesBasePath}/workload-cves/image/:imageId`;
+export const vulnerabilitiesWorkloadCveDeploymentSinglePath = `${vulnerabilitiesBasePath}/workload-cves/deployment/:deploymentId`;
+
```

Comment:
What are pro and con of singular versus plural nouns?

Although classic Vulnerability Management used it to distinguish independent pages from side panels, this feature has a simpler information architecture, correct?

Other containers like collections use plural nouns analogous to REST convention:
* /main/collections for list page
* /main/collections/:collectionIid for individual page

Reply:
Correct, I'm not sure why I subconsciously keep gravitating towards the singular version. Agreed that plural is more consistent too, I'll update.